packages/ra-core/src/controller/field/useReferenceManyFieldController.ts

```diff
@@ -26,6 +26,7 @@ export interface UseReferenceManyFieldControllerParams<
     sort?: SortPayload;
     source?: string;
     target: string;
+    queryOptions?: { meta?: any };
```

Comment:
Bad type. This should be ` UseGetManyReferenceHookOptions<RecordType>`

Reply:
Do you mean that I should change it like this?
```javascript
export interface UseGetManyReferenceHookOptions<RecordType> {
  meta?: any
}
...
  queryOptions?: UseGetManyReferenceHookOptions<RecordType>
```

This is the type of `queryOptions` in `ReferenceOneField.tsx`, which I used as a reference
```javascript
queryOptions?: UseQueryOptions<{
        data: ReferenceRecordType[];
        total: number;
    }> & { meta?: any };
```

Reply:
@Nela62 

Indeed there is no `UseGetManyReferenceHookOptions`. But I believe what @fzaninotto means is that you should use the same type as what `useGetManyReference()` expects, plus `meta`, i.e.:

```ts
queryOptions?: UseQueryOptions<{ data: RecordType[]; total: number }, Error> & { meta?: any };
```